SQL/New_patches/2018_07_23_battery_manager_permissions.sql

```diff
@@ -0,0 +1,39 @@
+-- Add Battery Manager to LorisMenu
+INSERT INTO LorisMenu (Parent, Label, Link, Visible, OrderNumber)
+       VALUES (
+           (SELECT ID FROM (SELECT ID FROM LorisMenu WHERE Label = 'Admin') as lm),
```

Comment:
```suggestion
           (SELECT ID FROM LorisMenu WHERE Label = 'Admin'),
```

I'm assuming this is a typo?

Reply:
Ah, I it's meant to be a subquery in which I forgot a closing parenthesis?
`(SELECT ID FROM (SELECT ID FROM LorisMenu) WHERE Label = 'Admin') as lm)`
From what I remember this has to be done because you can't select direct from a table you are currently inserting into, so I subquery has to be made or else it will throw an error.